contrib/automation_tests/test_cases/top_down_tab.py

```diff
@@ -119,7 +119,40 @@ def _verify_rows_when_thread_expanded(self, tree_view_table):
             ["OrbitTest (all threads)", "100.00% (1583)", "", "", "", ""],
             ["OrbitThread_203 [20301]", "79.34% (1256)", "", "", "", ""],
             ["clone", "79.22% (1254)", "0.00% (0)", "99.84%", "libc-2.24.so", "0x7fe86eff5900"],
-            ["[unwind errors]", "0.13% (2)", "", "0.16%", "", ""],
+            ["[Unwind errors]", "0.13% (2)", "", "0.16%", "", ""],
+        ]
+        self._verify_first_rows(tree_view_table, expectations)
+        logging.info("Verified content of top-down view when one thread is expanded")
+
+    def _verify_unwinding_errors_when_thread_expanded(self, tree_view_table):
+        logging.info("Expanding a thread of the top-down view")
```

Comment:
This is wrong, you are expanding the "[Unwind errors]" node here.

Reply:
Arg. Thanks!